libz-rs-sys-cdylib/src/gz.rs

```diff
@@ -186,7 +189,9 @@ enum Source {
 ///
 /// The caller must ensure that `path` and `mode` point to valid C strings. If the
 /// return value is non-NULL, caller must delete it using only [`gzclose`].
-#[cfg_attr(feature = "export-symbols", export_name = crate::prefix!(gzopen))]
+///
+/// [`gzfree`]: crate::z_stream
+#[export_name = crate::prefix!(gzopen)]
```

Comment:
I think this will have to stay mangled during testing to avoid overwriting the C versions of these functions.

Reply:
The test crate uses both the C and Rust versions of these symbols, right?

Reply:
Oh, right. I'm now setting the `semver-prefix` feature for `libz-rs-sys-cdylib` in the `test-libz-rs-sys` `Cargo.toml`, which will mangle the names when testing so that they don't clash with the C symbols.